gr-fec/lib/polar_decoder_common.cc

```diff
@@ -167,12 +166,13 @@ void polar_decoder_common::extract_info_bits(unsigned char* output,
 void polar_decoder_common::print_pretty_llr_vector(const float* llr_vec) const
 {
     for (int row = 0; row < block_size(); row++) {
-        // FIXME this is an interesting mixture of iostream and stdio
-        std::cout << row << "->" << int(bit_reverse(row, block_power())) << ":\t";
+        std::ostringstream msg;
+        msg << row << "->" << int(bit_reverse(row, block_power())) << ":\t";
```

Comment:
For consistency this could have been a `boost::format` aswell, but i suppose that's optional.